caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters to run online motion correction followed by online CNMF 
```

Comment:
Feels like a run-on sentence; rephrase this somehow?

Reply:
edited to be clearer